src/stores/chatExtras.js

```diff
@@ -130,6 +131,18 @@ export const useChatExtrasStore = defineStore('chatExtras', {
 			Vue.delete(this.parentToReply, token)
 		},
 
+		/**
+		 * Restore chat input from the browser storage and save to store
+		 *
+		 * @param {string} token The conversation token
+		 */
+		restoreChatInput(token) {
+			const chatInput = BrowserStorage.getItem('chatInput_' + token)
+			if (chatInput) {
+				Vue.set(this.chatInput, token, chatInput)
+			}
```

Comment:
Otherwise you're removing the stored value before we get them
```js
		getChatInput(token) { // Move from getters to action
			if (this.chatInput[token] === undefined) {
				this.restoreChatInput(token)
			}
			return this.chatInput[token]
		},
```
```suggestion
			Vue.set(this.chatInput, token, chatInput ?? '')
```